cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -68,6 +69,36 @@ function LogViewerController ($scope, LogViewerStore) {
     }
   };
 
+  myLogsApi.nextLogsJson({
+    'namespace' : this.namespaceId,
+    'appId' : this.appId,
+    'programType' : this.programType,
+    'programId' : this.programId,
+    'runId' : this.runId
+  }).$promise.then(
+    (res) => {
+      angular.forEach(res, (element, index) => {
+        if(res[index].log.logLevel === 'WARN'){
+          this.warningCount++;
+        } else if(res[index].log.logLevel === 'ERROR'){
+          this.errorCount++;
```

Comment:
This should be the case, however the current Timeline query is not working as expected